lib/src/main/java/graphql/nadel/engine/util/GraphQLUtil.kt

```diff
@@ -549,3 +552,19 @@ internal fun javaValueToAstValue(value: Any?): AnyAstValue {
 
 val GraphQLSchema.operationTypes
     get() = listOfNotNull(queryType, mutationType, subscriptionType)
+
+fun compileToDocument(
+    schema: GraphQLSchema,
+    operationKind: OperationDefinition.Operation,
+    operationName: String?,
+    topLevelFields: List<ExecutableNormalizedField>,
+    variablePredicate: VariablePredicate?,
+): CompilerResult {
+    return ExecutableNormalizedOperationToAstCompiler.compileToDocument(
+        schema,
+        operationKind,
+        operationName,
+        topLevelFields,
+        variablePredicate,
+    )
+}
```

Comment:
This is a weird helper - since there is no helping - just a flattening of the name

Reply:
Oh yeah just Kotlin-ifys it so we can use named parameters.